chainerio/containers/zip.py

```diff
@@ -98,7 +98,24 @@ def info(self):
 
     def stat(self, path):
         self._open_zip_file()
-        return self.zip_file_obj.getinfo(path)
+        try:
+            stat = self.zip_file_obj.getinfo(path)
+        except KeyError:
+            # handles cases when path is a directory but without trailing slash
+            # see issue $67
+            if path.endswith('/'):
```

Comment:
I don't think it's a good practice to write main logic codes in except clause for readability. There should only be error handling. Maybe directly using `NameToInfo` or just turning off the existence flag would be nicer.